.vscode/settings.json

```diff
@@ -23,4 +23,7 @@
       "autoFix": true
     },
   ],
+  "editor.codeActionsOnSave": {
```

Comment:
The ESLint plugin insisted on this change (added every time I removed it).

Reply:
interesting, I will keep it then